src/boot_fmt/core.clj

```diff
@@ -61,11 +70,13 @@ Specify the operation using the --mode parameter:
    f files VAL #{str} "The list of files or directories to format"
    s source bool "Automatically scan for files in boot source-paths and resource-paths"
    g git bool "Automatically scan for files in current git repository"
+   c cached bool "Automatically scan for cached files in current git repository"
    o options OPTS edn "zprint options"]
   (let [mode (or mode :print)
         files (cond-> files
                 source (clojure.set/union (find-files-source))
-                git (clojure.set/union (find-files-git)))]
+                git (clojure.set/union (find-files-git))
+                cached (clojure.set/union (find-files-git-cached)))]
```

Comment:
I think `git diff` would be useful as well as `git diff --cached`. Perhaps two options?

`-D --git-diff Automatically scan for changed files using git diff`

`-C -git-diff-cached Automatically scan for changed files using git diff --cached`